dll/win32/shell32/dialogs/fprop.cpp

```diff
@@ -137,8 +137,12 @@ SH_ShowPropertiesDialog(LPCWSTR pwszPath, LPCITEMIDLIST pidlFolder, PCUITEMID_CH
                 hr = pFileDefExt->AddPages(AddPropSheetPageCallback, (LPARAM)&Header);
                 if (FAILED(hr))
                     ERR("AddPages failed\n");
-            } else
+            }
+            else
+            {
                 ERR("Initialize failed\n");
```

Comment:
Please include the `hr` in the logging as hexadecimal.

Reply:
or use `!FAILED_UNEXPECTEDLY(hr)` instead of the `SUCCEEDED(hr)` for this case.

Reply:
Done